src/main/java/com/github/theholywaffle/teamspeak3/FileTransferHelper.java

```diff
@@ -55,7 +58,7 @@ public void downloadFile(OutputStream dataOut, FileTransferParameters params) th
 		final long dataLength = params.getFileSize();
 		final String prefix = "[Download " + (params.getClientTransferId() + 1) + "] ";
```

Comment:
Might be a good idea to either remove the space from the end of this String and add it between the `{}` and the log message to make this look a bit less awkward, or directly use the built-in string replacing à la

```
final int downloadId = params.getClientTransferId() + 1:
[...]
log.info("[Download {}] Very important log message", downloadId);
```

The first approach leads to less code duplication, the second one might be easier to read. Choose whichever you like better 😛